src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1908,17 +1909,23 @@ properties:
     $ref:
       - "_terms.yaml#/imaging_findings/common"
     enum:
+      - Bladder Involvement
       - Carcinomatosis
       - Distant Metastasis
       - Equivocal
+      - Extra-Pelvic Metastatic Disease
```

Comment:
what should the capitalization be like per our rules?
"Extra-pelvic Metastatic Disease" (Jira)
or "Extra-Pelvic Metastatic Disease" (the PR)

Reply:
Per the style guide:  "For hyphenated words, e.g. Iliac-External, all elements should be capitalized unless they are articles (a, an, the), prepositions (in, on, off, out, etc.) or coordinating conjunctions (and, but, for, or, nor)."

So yes, the PR is the correct one.  JIRA is now fixed.